foursight_core/react/api/react_api.py

```diff
@@ -356,10 +359,19 @@ def _reactapi_header_cache(self, request: dict, env: str) -> dict:
             },
             "versions": self._get_versions_object(),
             "portal": {
-                "url": app.core.get_portal_url(env or default_env),
+                "url": portal_url,
                 "health_url": portal_base_url + "/health?format=json",
                 "health_ui_url": portal_base_url + "/health"
             },
+            "resources": {
+                "es": app.core.host,
+                "foursight": self.foursight_instance_url(request),
+                "portal": portal_url,
+                "rds": os.environ["RDS_HOSTNAME"],
```

Comment:
If this is a public protocol and we later needed additional information, it might bet tricky to add. Should this key be "rds_hostname" or could the data be `"rds": {"hostname": os.environ["RDS_HOSTNAME"]}`? I'm wondering if we might later need to add either user name or one of the other things like that. I guess that's not a small change, but if we're going to make it, this would be the time. Feel free to push back. @willronchetti  might have an opinion.

Reply:
You mean so we can display the RDS username for info purposes? Not such a big change really, or could just add rds_username, either way, will add comment to this effect.